.github/workflows/tests.yml

```diff
@@ -13,9 +13,15 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        stack: [blade, react, react --ssr, vue, vue --ssr, api]
+        stack: [blade, react, vue, api]
+        args: [null, --pest]
+        include:
+          - stack: vue
+            args: --ssr
+          - stack: react
+            args: --ssr
 
-    name: Test Stubs (${{ matrix.stack }})
+    name: Test Stubs - ${{ matrix.stack }} ${{ matrix.args }}
```

Comment:
I changed the formatting here just so that when `matrix.args` is null, the whitespace before it is not obvious.